### PR TITLE
upgrade `jjwt` to latest version `0.11.1`

### PR DESCRIPTION
**Pull request by LiYing2010**
### Change List
upgrade `jjwt` to latest version `0.11.1`

this should fix issue #68

### CODE REVIEW

File: line-sdk/build.gradle [hunk 1]
```diff
@@ -82,10 +82,10 @@ dependencies {
     testImplementation 'org.robolectric:robolectric:4.3'
     testImplementation 'org.mockito:mockito-core:2.28.2'
 
-    api 'io.jsonwebtoken:jjwt-api:0.10.5'
-    implementation 'io.jsonwebtoken:jjwt-impl:0.10.5'
-    implementation('io.jsonwebtoken:jjwt-orgjson:0.10.5') {
-        exclude group: 'org.json', module: 'json' //provided by Android natively
+    api "io.jsonwebtoken:jjwt-api:0.11.1"
```
Comment (plateaukao) [inline line-sdk/build.gradle:85]: please use single quote as other lines.

File: line-sdk/build.gradle [hunk 2]
```diff
@@ -82,10 +82,10 @@ dependencies {
...
+    implementation "io.jsonwebtoken:jjwt-impl:0.11.1"
```
Comment (plateaukao) [inline line-sdk/build.gradle:86]: ditto for other lines.

File: line-sdk/build.gradle (hunk 1 above)
Comment (LiYing2010) [reply line-sdk/build.gradle]: extracted jjwt_version  as a property in build script

File: line-sdk/build.gradle (hunk 2 above)
Comment (LiYing2010) [reply line-sdk/build.gradle]: extracted `jjwt_version`  as a property in build script

**Pull request merged**